packages/docs/docs/guides/8-glossary.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Glossary
+keywords:
+---
+
+# Ecosystem
+
+## Request Portal API
+
+The Request Portal is a software built on top of the Request Protocol that allows third party software to easily integrate Request, all the private keys to create requests and perform actions to them are stored by Request and can be managed through OAuth. The Request Portal API is an API built on top of the Request Portal.
+
+## Request Client
+
+The Request Client is a Javascript library made to interact directly with the Request Protocol. The Request Client connects to a Request Node.
+
+## Request Node
+
+Request Nodes are HTTP servers exposing an API used to allow Request Client to communicate with the Request Protocol. These servers abstract the complexity of IPFS and Ethereum used by the current implementation of the Request Protocol.
+
+## Request Protocol
+
+The Request Protocol is the underlying protocol to make Request work. The protocol stores all the request information and actions performed to them on a distributed ledger.
+
+# Request Protocol
+
+## Action
+
+An action is signed data added by a request's stakeholder into the Request Protocol that creates or updates the state of a request. The creation of a request is considered as an action and a request can be represented by a list of actions.
+
+## Balance
+
+When using a payment network, the balance is the current amount paid for a request. This balance is not the current balance of a specific account but is determined by the payment detection method of the payment network used.
+
+A request with no payment network provided doesn't have a balance.
+
+## Confirmed/Pending action
+
+The Protocol relies on other blockchain technologies to ensure data immutability. Most blockchain doesn't offer transaction instant finality. This means that when performing an action on the request, this action can't directly be confirmed as effective.
```

Comment:
This is a good explanation of why we need that, but in my opinion doesn't explain _what_ it is, and how to use it. I would add something like: "Pending state is useful to have a fast response, and good user experience. Until the request is Confirmed, it should not be relied upon"

Also, avoid refering to Request as "the Protocol"